bin/_mocha

```diff
@@ -187,13 +187,44 @@ Error.stackTraceLimit = Infinity; // TODO: config
 
 var reporterOptions = {};
 if (program.reporterOptions !== undefined) {
-    program.reporterOptions.split(",").forEach(function(opt) {
-        var L = opt.split("=");
-        if (L.length != 2) {
-            throw new Error("invalid reporter option '" + opt + "'");
+
+   program.reporterOptions.split(",").forEach(
+
+    // Two possible formats ...
+     /\w+{(\w+:\w+)+}/.test(program.reporterOptions) ?
```

Comment:
@benvinegar You could add an integration test: https://github.com/mochajs/mocha/tree/master/test/integration The helpers allow you to invoke the mocha binary (and hence _mocha as well) https://github.com/mochajs/mocha/blob/master/test/integration/helpers.js#L25